config/src/config/consensus_config.rs

```diff
@@ -136,7 +136,7 @@ pub struct PipelineBackpressureValues {
     // If we want to dynamically increase it beyond quorum_store_poll_time,
     // we need to adjust timeouts other nodes use for the backpressured round.
     pub backpressure_proposal_delay_ms: u64,
-    pub max_txns_from_block_to_execute: Option<usize>,
+    pub max_txns_from_block_to_execute: Option<u64>,
```

Comment:
Just to be sure, this value is only used in the config right? (Not sent over wire)

Reply:
it is sent over the wire! that's what I am fixing :D 

usize will have different width whether compiled on 32bit or 64bit machine, and so is broken for being used over the wire. luckily all validators use 64bit machine, so this only works today because it is effectively u64 - so changing it to that.

Reply:
wow, ok. I wonder if there's a way to have the serializer reject usize, or some other lint/compile time checks